ui/raidboss/data/04-sb/ultimate/ultima_weapon_ultimate.ts

```diff
@@ -67,6 +85,7 @@ export interface Data extends RaidbossData {
   ifritUntargetableCount: number;
   titanGaols: string[];
   seenTitanGaols?: boolean;
+  lastTitanMove?: NetMatches['ActorMove'];
```

Comment:
As I mentioned in my other comment, there's a chance that if the tank is incredibly precise, keeping Titan from spinning at all, and Titan chooses to jump north, there will not be an `ActorMove` packet for Titan at the start of the phase. Maybe it makes sense to default this property to a default "middle of arena, facing north" packet, or maybe it makes sense to remove the `undefined` check from the `UWU Titan Jump Direction` and then return a default of north if it's undefined.

Reply:
Interesting edge case, but makes sense.  I think it probably makes sense to constrain this to just the first jump for added safety.  Let me know if this approach works.